packages/vitest/src/node/cli/cli-config.ts

```diff
@@ -801,6 +801,11 @@ export const cliOptionsConfig: VitestCLIOptions = {
     description:
       'Clear terminal screen when re-running tests during watch mode (default: `true`)',
   },
+  configLoader: {
+    description:
+      'Use `bundle` to bundle the config with esbuild or `runner` (experimental) to process it on the fly (default: `bundle`)',
+    argument: '<loader>',
+  },
```

Comment:
documentation should also mention this is only available from newer Vite version.